README.md

```diff
@@ -1,39 +1,154 @@
 # Wavefront by VMware SDK for C# [![travis build status](https://travis-ci.com/wavefrontHQ/wavefront-sdk-csharp.svg?branch=master)](https://travis-ci.com/wavefrontHQ/wavefront-sdk-csharp) [![NuGet](https://img.shields.io/nuget/v/Wavefront.SDK.CSharp.svg)](https://www.nuget.org/packages/Wavefront.SDK.CSharp)
 
-Wavefront by VMware SDK for C# is the core library for sending metrics, histograms and trace data from your .NET application to Wavefront using an `IWavefrontSender` interface.
-
-## Frameworks Supported
+## Table of Content
+* [Prerequisites](#Prerequisites)
+* [Set Up an IWavefrontSender](#set-up-an-iwavefrontsender)
+* [Send Data to Wavefront](#send-data-to-wavefront)
+* [Close the IWavefrontSender](#close-the-iwavefrontsender)
+* [License](#License)
+* [How to Get Support and Contribute](#how-to-get-support-and-contribute)
+
+# Welcome to the Wavefront C# SDK
+
+Wavefront by VMware C# SDK lets you send raw data from your .NET application to Wavefront an `IWavefrontSender` interface. The data is then stored as metrics, histograms, and trace data. This SDK is also called the Wavefront Sender SDK for C#. 
+
+Although this library is mostly used by the other Wavefront C# SDKs to send data to Wavefront, you can also use this SDK directly. For example, you can send data directly from a data store or CSV file to Wavefront.
+
+**Before you start implementing, let us make sure you are using the correct SDK!**
+
+![C# Sender SDK Decision Tree](docs/csharp_sender_sdk.png)
+
+> ***Note***:
+> </br>
+>   * **This is the Wavefront by VMware SDK for C# (Wavefront Sender SDK for C#)!**
+>   If this SDK is not what you were looking for, see the [table](#wavefront-sdks) below.
+
+
+#### Wavefront SDKs
```

Comment:
Should we start removing the Framework SDKs? Or only from doc?

Reply:
We thought we will leave the framework SDKs on the READMEs for now. But Once we remove it from docs we will need to think about some broken links here. :)